crates/core/src/operations/create.rs

```diff
@@ -233,8 +237,36 @@ impl CreateBuilder {
             )
         };
 
+        let mut converted_writer_features = self
+            .configuration
+            .iter()
+            .map(|(key, _)| key.clone().into())
+            .filter(|v| match v {
+                WriterFeatures::Other(_) => false,
+                _ => true,
+            })
+            .collect::<HashSet<WriterFeatures>>();
+
+        let mut converted_reader_features = self
+            .configuration
+            .iter()
+            .map(|(key, _)| key.clone().into())
+            .filter(|v| match v {
+                ReaderFeatures::Other(_) => false,
+                _ => true,
+            })
+            .collect::<HashSet<ReaderFeatures>>();
+
+        let contains_timestampntz = &self
+            .columns
+            .iter()
+            .any(|f| f.data_type() == &DataType::TIMESTAMPNTZ);
```

Comment:
Should the enum variant be all caps here?

Reply:
This points to the pub const on DataType:

`pub const TIMESTAMPNTZ: Self = DataType::Primitive(PrimitiveType::TimestampNtz);`

Reply:
missed this, never mind :-D